beacon_node/beacon_chain/src/migrate.rs

```diff
@@ -676,6 +676,7 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> BackgroundMigrator<E, Ho
                     StoreOp::DeleteBlock(block_root),
                     StoreOp::DeleteExecutionPayload(block_root),
                     StoreOp::DeleteBlobs(block_root),
```

Comment:
This prunes all non-canonical blocks. You should also prune finalized canonical non-checkpoint sync committee branches

Reply:
I am now pruning canonical non-checkpoint sync committee branches